packages/core/template/default/index.md

```diff
@@ -1,132 +1,75 @@
-</modal>
+This _default_ site comes pre-configured with the core <a href="https://markbind.org/userGuide/components/navigation.html#navigation-components" target="_blank">Navigation components</a>: a <tooltip content="Site Navigation">**siteNav**</tooltip>, a <tooltip content="Page Navigation">**pageNav**</tooltip>, a <tooltip content="Navigation Bar">**NavBar**</tooltip>, and a **Search Bar**</tooltip>. To help you get started with the **siteNav**, we have included <tooltip content="Topic 1, Topic 2, Topic 3, Topic 3a, Topic 3b">five dummy placeholder pages</tooltip>. The **NavBar** also comes with a placeholder slot for your custom Logo.
 
-**A table:**
+---
 
-Column 1 | Column 2 | Column 3 | :far-thumbs-up: / :far-thumbs-down:?
-:------: | :------: | :------: | ----
-value1   | x        | 5        | :far-thumbs-up:
-value2   | y        | 20       | :far-thumbs-down:
+## Guide to MarkBind
 
+To see the capability of MarkBind in action, feel free to take a look at some of the websites built using MarkBind on our <a href="https://markbind.org/userGuide/gettingStarted.html" target="_blank">Showcase</a> page. 
 
-## Sub Heading 1.2
+For more information on how to work with MarkBind sites and to add content, refer to our comprehensive <a href="https://markbind.org/userGuide/gettingStarted.html" target="_blank">User Guide</a>. 
 
-**Media embeds:**
+<box type="info">
 
-@[youtube](http://www.youtube.com/watch?v=v40b3ExbM0c)
+If you are interested in contributing to MarkBind, you can refer to our <a href="https://markbind.org/devdocs/devGuide/devGuide.html" target="_blank">Developer Guide</a> as well!
 
-**Tabs:**
+</box>
 
-<tabs>
-  <tab header="Tab X">
-    Some text some text some text some text some text some text some text. Some text some text some text some text some text some text some text. Some text some text some text some text some text some text some text some text some text some text some text some text some text some text. Some text some text some text some text some text some text. Some text some text some text some text some text some text some text.
-  </tab>
-  <tab header="Tab Y">
-    ...
-  </tab>
-  <tab-group header="Tab group">
-    <tab header="Tab Y.1">
-      ...
-    </tab>
-    <tab header="Tab Y.2">
-      ...
-    </tab>
-  </tab-group>
-</tabs>
+<panel header="**Good starting points in our User Guide:**" expanded no-close>
 
-<br>
+##### **User Guide: Authoring Contents**
 
-# Heading 2
+> Learn about the variety of syntax schemes, formats, and custom MarkBind components that you can use in your MarkBind site.
 
-**Some boxes:**
+More info in: _<a href="https://markbind.org/userGuide/authoringContents.html" target="_blank">User Guide → Authoring Contents</a>_
```

Comment:
This link and line 65 is the same, is this a mistake?

Reply:
Fixed in [945ea73](https://github.com/MarkBind/markbind/pull/2225/commits/945ea738078da9000f4299903da634c4db9dee5e)!